examples/launch_templates/main.tf

```diff
@@ -73,5 +73,14 @@ module "eks" {
       public_ip                     = true
       elastic_inference_accelerator = "eia2.medium"
     },
+    {
+      name                   = "worker-group-4"
+      instance_type          = "t3.small"
+      asg_desired_capacity   = 1
+      public_ip              = true
+      root_volume_size       = 150
+      root_volume_type       = "gp3"
+      root_volume_throughput = 300
+    },
```

Comment:
@antonbabenko From what I understood with your first comment, you are expecting an example like this at `examples/launch_templates/main.tf`, right? Please correct me if I misunderstood.

> I don't see changes for throughput inside of ebs_block_device block [here](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1445/files#diff-abc45b07ed4bab6d21313623302e9685703e3b4ad4a981fd57fe91c4c70095edR315)

The 'here' link somehow only show me the PR diff page, and isn't highlighting anything.

Reply:
The link works only if you expand the PR a bit.

I meant that `dynamic "ebs_block_device"` block on line 315 has to have a `throughput` in content, too.

Once there is a `throughput` in `ebs_block_device`, update file `examples/launch_templates/main.tf` and add `additional_ebs_volumes` with some data including the new argument you add.

Reply:
Thanks @antonbabenko, fixed now.

For launch templates, the config is already there: https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/workers_launch_template.tf#L505